docs/en/getting_started.md

```diff
@@ -54,24 +71,143 @@ many workers were created when you started faucet. This is because faucet
 automatically detects the number of CPU cores on your computer and creates
 a worker for each core.
 
+To know how many CPU cores you have, you can run the following commands:
+
+- On Linux:
+```bash
+lscpu
+```
+
+- On Windows Powershell
+```bash
+Get-WmiObject -Class Win32_Processor | Select-Object NumberOfCores, NumberOfLogicalProcessors
+```
+
 You can customize the number of workers by using the `--workers` flag:
 
 ```bash
-faucet --workers 4
+faucet start --workers 4
 ```
 
 Or by setting the `FAUCET_WORKERS` environment variable:
 
 ```bash
 export FAUCET_WORKERS=4
-faucet
+faucet start
 ```
 
 In both cases, faucet will create 4 workers on random available ports.
 Traffic will be load balanced across all workers according to the
 IP address of the incoming request. This means that if you have 4 workers,
 then you can handle 4 times as many concurrent requests as a single worker.
 
+### Router Mode
+
+**When to use Router?**
+
+- **Multiple Applications:** Use Router mode when you need to deploy and manage multiple applications on different routes but on the same port.
+
+- **Centralized Management:** If you desire a centralized configuration to route requests to the corresponding applications based on the route, Router is the appropriate option.
+
+- **Resource Optimization:** Router facilitates the management and scalability of various applications by allowing an efficient distribution of requests.
+
+To start faucet in Router mode, we first need a configuration file where the router logic `frouter.toml` will be placed. The configuration file must be in the root of the repository.
```

Comment:
This is not necesarilly true, it should be at the root of the working directory where you want to run the applications. 

For example, if we have a directory structure that has a directory called, "apps", the router could easily by in said directory as long as the working directory is "apps".

In this case, not every person using faucet will be using a repo, so just calling it working directory should be best.